### PR TITLE
Adding Augmentor

### PR DESCRIPTION
**Pull request by claydugo**
Adds [Augmentor](https://pypi.org/project/Augmentor/). Generated using `grayskull`

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by claydugo** [comment]
@conda-forge/help-python, this is ready

File: recipes/Augmentor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pillow >=5.2.0
+    - tqdm >=4.9.0
+    - numpy >=1.11.0
+    - futures >=3.2.0  # [py==27]
```
Comment (ocefpaf) [inline recipes/Augmentor/meta.yaml:25]: We don't  build py27, when removing this you can make this package noarch.

```suggestion
```

File: recipes/Augmentor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/Augmentor/meta.yaml:21]:
```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
```

File: recipes/Augmentor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Augmentor-{{ version }}.tar.gz
+  sha256: 1198672f1ab9f97f1e1802803b9165ccb4830e30f8b5f324a7bb2f8f38331d1e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```
Comment (ocefpaf) [inline recipes/Augmentor/meta.yaml:13]:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Comment by claydugo** [comment]
Thank you for review, these should be addressed correctly now

**Comment by ocefpaf** [comment]
> Thank you for review, these should be addressed correctly now

See https://github.com/conda-forge/staged-recipes/pull/24365#pullrequestreview-1700538689

PS: If you commit the suggestion you can avoid errors.

**Comment by claydugo** [comment]
> PS: If you commit the suggestion you can avoid errors.

Got it, sorry about that.

**Pull request merged**